lib/chef/knife/bootstrap.rb

```diff
@@ -313,7 +313,7 @@ def run
 
         # chef-vault integration must use the new client-side hawtness, otherwise to use the
         # new client-side hawtness, just delete your validation key.
-        if chef_vault_handler.doing_chef_vault? || !File.exist?(File.expand_path(Chef::Config[:validation_key]))
+        if chef_vault_handler.doing_chef_vault? || Chef::Config[:validation_key] && !File.exist?(File.expand_path(Chef::Config[:validation_key]))
```

Comment:
i like parens around the `&&` expression written like this so i don't have to think about if operator precedence matters.